src/index.ts

```diff
@@ -10,31 +10,8 @@ import { PLUGIN_NAME, ABI_SUFFIX, DEFAULT_STARKNET_SOURCES_PATH, DEFAULT_STARKNE
 import { HardhatConfig, HardhatRuntimeEnvironment, HardhatUserConfig, HttpNetworkConfig } from "hardhat/types";
 import { adaptLog, adaptUrl, getDefaultHttpNetworkConfig } from "./utils";
 import { DockerWrapper, VenvWrapper } from "./starknet-wrappers";
+import { glob } from "hardhat/internal/util/glob";
```

Comment:
`glob` should be directly imported, not depending on it being in hardhat internals.

Reply:
After the call, we mentioned potential use of `util.promisify ` or something like that.